packages/core/src/__tests__/identity-permissions.test.ts

```
@@ -0,0 +1,595 @@
+
+  it("refuses an agent handing out identity:configure even when it holds it (KTD20)", () => {
+    const grantorGrants = grantsOf(
+      { permission: "roles:grant", disposition: "allow" },
+      { permission: "identity:configure", disposition: "allow" },
+    );
+    expect(
+      evaluateGrantAuthority({
+        grantorGrants,
+        grantorKind: "agent",
+        targetActorId: AGENT.id,
+        targetKind: "agent",
```

Biome (2.5.6)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)